### DISCUSSION TITLE
Impose grain orientation

### DISCUSSION THREAD

**Discussion by hchaieb1234** [mtex-toolbox/mtex#567] [Ask Anything]
Hello,

Could you please tell me how to impose an orientation of a grain having the ID X to be the same as a grain having an ID Y?

Thank you.

**Comment by kilir** [comment]
Hi,
```
grains('id',X).meanOrientation= grains('id',Y).meanOrientation
```
should do it.
Cheers,
Rüdiger

**Reply by hchaieb1234** [reply]
Thank you but this does not affect the EBSD IPF map, I want to impose the orientation of twins to be the same as the mother grain and renconstruct the IPF map (as if there were no twins)

**Reply by kilir** [reply]
Of course this does not change the orientations of each EBSD point as you were asking for grains.

It seems you want to do something else. If you know your twinning relationship, you can take all orientations in one grain and rotate them to the untwinned orientation - depending on the twinning relation this might be straight forward or not.
Have a look in the documentation https://mtex-toolbox.github.io/Documentation.html under "Phase Transitions", maybe you find what you want there.
Cheers,
Rüdiger
